hack/asciidoc-flows-gen.sh

```diff
@@ -1,27 +1,33 @@
 #!/bin/sh
 
 ADOC=docs/flows-format.adoc
-SOURCE=controllers/consoleplugin/config/static-frontend-config.yaml
+FE_SOURCE=controllers/consoleplugin/config/static-frontend-config.yaml
+LOKI_LABEL_SOURCE=pkg/helper/loki/loki-labels.json
+CARDINALITY_SOURCE=pkg/helper/cardinality/cardinality.json
+OTEL_SOURCE=pkg/helper/otel/otel-config.json
 
 # Header
 echo "// Automatically generated by '$0'. Do not edit, or make the NETOBSERV team aware of the editions." > $ADOC
 cat ./hack/flows-format-header.adoc >> $ADOC
 echo -e "\n" >> $ADOC
 
-echo -e '[cols="1,1,3,1,1,1",options="header"]' >> $ADOC
+echo -e '[cols="1,1,3,1,1,1,1",options="header"]' >> $ADOC
 echo -e '|===' >> $ADOC
-echo -e '| Name | Type | Description | Filter ID | Loki label | Cardinality' >> $ADOC
+echo -e '| Name | Type | Description | Filter ID | Loki label | Cardinality | Otel' >> $ADOC
```

Comment:
Since this ends up in the downstream doc we should write the official name and not "otel"

```suggestion
echo -e '| Name | Type | Description | Filter ID | Loki label | Cardinality | OpenTelemetry' >> $ADOC
```

Reply:
+1